Common.mk

```diff
@@ -177,6 +177,23 @@ BUILD_OCI_TARS?=false
 LOCAL_IMAGE_TARGETS=$(foreach image,$(IMAGE_NAMES),$(image)/images/amd64) $(if $(filter true,$(HAS_HELM_CHART)),helm/build,) 
 IMAGE_TARGETS=$(foreach image,$(IMAGE_NAMES),$(if $(filter true,$(BUILD_OCI_TARS)),$(call IMAGE_TARGETS_FOR_NAME,$(image)),$(image)/images/push)) $(if $(filter true,$(HAS_HELM_CHART)),helm/push,) 
 
+
+############# WINDOWS #############################
+# similiar to https://github.com/kubernetes-csi/livenessprobe/blob/master/release-tools/prow.sh#L78
+WINDOWS_IMAGE_REGISTRY=mcr.microsoft.com/windows
+WINDOWS_BASE_IMAGE_NAME=nanoserver
+WINDOWS_ADDON_IMAGE_NAME=servercore
```

Comment:
I assume we're going to want to stay up to date on these and watch for vulnerabilities like we do for our other base image dependencies.